src/matchers/toEqualText/index.test.ts

```diff
@@ -10,71 +11,72 @@ describe("toEqualText", () => {
     it("positive frame", async () => {
       await page.setContent(`<iframe src="https://example.com"></iframe>`)
       const iframe = await page.$("iframe")
-      const result = await toEqualText(iframe!, "h1", "Example Domain")
-      expect(result.pass).toBe(true)
-      expect(result.message()).toMatchSnapshot()
+      await expect(iframe!).toEqualText("h1", "Example Domain")
     })
     it("positive", async () => {
       await page.setContent(`<div id="foobar">Bar</div>`)
-      const result = await toEqualText(page, "#foobar", "Bar")
-      expect(result.pass).toBe(true)
-      expect(result.message()).toMatchSnapshot()
+      await expect(page).toEqualText("#foobar", "Bar")
     })
     it("negative", async () => {
       await page.setContent(`<div id="foobar">zzzBarzzz</div>`)
-      expect(
-        testWrapper(await toEqualText(page, "#foobar", "not-existing"))
-      ).toThrowErrorMatchingSnapshot()
+      await assertSnapshot(() => expect(page).toEqualText("#foobar", "Bar"))
+    })
+    describe("not", () => {
+      it("success in frame", async () => {
```

Comment:
Suggestions on the naming of these tests?  Previously there weren't any tests for `expect(page).not...` but when I added one, the naming of positive/negative in the test names is a bit confusing.

Reply:
@mskelton idk, maybe `with 'not' usage` or smth like this